Annotato/Annotato/Persistence/LocalPersistenceManager/CoreDataEntities/Properties/AnnotationEntity+CoreDataProperties.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import CoreData
+
+extension AnnotationEntity {
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<AnnotationEntity> {
+        NSFetchRequest<AnnotationEntity>(entityName: "AnnotationEntity")
+    }
+
+    @NSManaged public var id: UUID
+    @NSManaged public var width: Double
+    @NSManaged public var ownerId: String
+    @NSManaged public var originX: Double
+    @NSManaged public var originY: Double
+
+    // Relations
```

Comment:
I think we can do without these comments as they don't add much value, and we can tell the distinction using the line break.